### PR TITLE
Project 2 : Add / Remove

### PR DESCRIPTION
**Pull request by SamanAtashi**
# Project 2 : Add / Remove:
Features introduced are as follows:
1. Redux state management - reducers/actions
2. Connecting React app to Redux

### CODE REVIEW

File: src/components/Books/Book/Book.jsx [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
-    {book.name}
-    <button type="button">remove</button>
-  </li>
-));
+const Book = () => {
+  const dispatch = useDispatch();
+  const removeBookFromStore = () => {
+    // dispatch(removeBook('newBook'));
+  };
+
```
Comment (DeliceLydia) [inline src/components/Books/Book/Book.jsx]: - [x] Please work on the `remove` feature I can see that you started from these codes but removeBook reducer is commented work on that and always remember to remove commented codes

File: src/components/Books/Book/AddBook.jsx [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+
+  const submitBookToStore = () => {
+    const newBook = {
+      id: uuidv4(),
+      title: document.getElementById('title').value,
+      author: document.getElementById('author').value,
+    };
+    // dispatch an action and pass it the newBook object (your action's payload)
+    dispatch(addBook(newBook));
+  };
```
Comment (DeliceLydia) [inline src/components/Books/Book/AddBook.jsx]: - [x] Work on the `add book` feature I can't add a book now please fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @SamanAtashi, 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇 
#### Hello @SamanAtashi , 

Your project is complete!  You have met all requirements for this project. It is a thumbs up for me. 👍🏽 You can go ahead and merge this PR. 
Congratulations! 🎉 👏👏

_Good luck with your next project and Happy coding!_ 💻

**Pull request merged**